examples/README.md

```diff
@@ -3,5 +3,6 @@
 Below are some examples showing some features of pbandk:
 
 * [gradle-and-jvm](gradle-and-jvm) - Shows how to use the Protobuf Plugin for Gradle
+* [gradle-and-jvm-no-json](gradle-and-jvm-no-json) - Shows how to use the Protobuf Plugin for Gradle without json serialization support
```

Comment:
I'd prefer to not add an entire example just for this option. There's a cost to maintaining the examples and updating them with changes to `pbandk`. I'm definitely on board for new examples that demonstrate some substantial piece of functionality. But for something simple like a `protoc-gen-kotlin` flag, I would prefer to just add it to the existing `gradle-and-jvm` example (maybe commented out?).